Backpack/build.gradle

```diff
@@ -1,9 +1,10 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'org.jetbrains.dokka-android'
+apply plugin: 'com.github.dcendents.android-maven'
 
 version = rootProject.ext.version
-group='com.github.Skyscanner.backpack-android'
+group='com.github.skyscanner.backpack-android'
```

Comment:
Why did this change from:
```
net.skyscanner.backpack:Backpack
```
to
```
com.github.Skyscanner:backpack-android
```

Reply:
Moved from `Bintray` to `jitpack`. see https://gojira.skyscanner.net/browse/BPK-1858

Reply:
Bu why does the names space need to change?

Reply:
this is how the repository is located by jitpack.
 ` implementation 'com.github.User:Repo:Version'  `

Reply:
Gotcha 👍